requirements/minimal.txt

```diff
@@ -1,11 +1,12 @@
 requests~=2.26
 mycroft-messagebus-client~=0.9,!=0.9.2,!=0.9.3
 combo-lock~=0.2
-ovos-utils~=0.0, >=0.0.26a2
-ovos-plugin-manager~=0.0, >=0.0.19a4
-ovos-config~=0.0,>=0.0.5a5
+ovos-utils~=0.0, >=0.0.26
+ovos-plugin-manager~=0.0, >=0.0.20a1
+# TODO: Stable OPM Release
```

Comment:
want to wait for this or merge as is? i think this can be bumped in #220 after rebasing

Reply:
I guess as-is since this still needs validation